lib/style.css

```diff
@@ -69,11 +79,15 @@ li div.newcomment button {
 }
 
 #new textarea {
-    max-width: 79em;
+    max-width: 1000px;
```

Comment:
Why this change? This switches from a 79 char width (which is a pretty common width for text) to an arbitrary pixel width depending on the font.